site/en/docs/extensions/mv3/override/index.md

```diff
@@ -73,26 +69,24 @@ For `PAGE_TO_OVERRIDE`, substitute one of the following:
 - `history`
 - `newtab`
 
-## Tips {: #tips }
+## Best practices {: #tips }
 
-For an effective override page, follow these guidelines:
+* **Make your page quick and small.** <br/>Users expect built-in browser pages to open instantly. Avoid doing things that might take a long
+time, like synchronous fetches of network or database resources.
```

Comment:
It really depends what you want, but two options are IndexedDB and chrome.storage.

Reply:
I didn't phrase the question well. Is it still possible to access data synchronously in extensions? Is there anything we need to direct people away from?

Reply:
I can't immediately think of anything.

Reply:
> @oliverdunk I want to specifically direct folks to use `fetch()` instead of `XMLHttpRequest()`.

How does this relate to the "synchronous fetches" line? That sounds similar enough to using `fetch()` that readers might get confused.

Reply:
My brain wasn't working that day. You can ignore it.

Reply:
I wrote a suggestion that resolves the confusion.

Reply:
thanks!